packages/land/test/mainnet/land.test.ts

```diff
@@ -931,6 +900,69 @@ describe('Land.sol', function () {
       ).not.to.be.reverted;
     });
   });
+
+  it('should return royaltyManager address', async function () {
+    const {LandContract, manager} = await loadFixture(setupLand);
+    expect(await LandContract.getRoyaltyManager()).to.be.equal(
+      await manager.getAddress(),
+    );
+  });
+
+  it('should not set royaltyManager if caller is not admin', async function () {
+    const {LandAsOther, other} = await loadFixture(setupLand);
+    await expect(
+      LandAsOther.setRoyaltyManager(await other.getAddress()),
+    ).to.be.revertedWith('only admin allowed');
+  });
+
+  it('should emit RoyaltyManagerSet event', async function () {
+    const {LandAsAdmin, other} = await loadFixture(setupLand);
+    const tx = await LandAsAdmin.setRoyaltyManager(await other.getAddress());
+    await expect(tx)
+      .to.emit(LandAsAdmin, 'RoyaltyManagerSet')
+      .withArgs(await other.getAddress());
+  });
+
+  it('should set royaltyManager', async function () {
+    const {LandAsAdmin, other, manager} = await loadFixture(setupLand);
+    expect(await LandAsAdmin.getRoyaltyManager()).to.be.equal(
+      await manager.getAddress(),
+    );
+    await LandAsAdmin.setRoyaltyManager(await other.getAddress());
+    expect(await LandAsAdmin.getRoyaltyManager()).to.be.equal(
+      await other.getAddress(),
```

Comment:
You don't need `.getAddress()` the new mattches support addressable, you can do: `equal(other)` it is clearer.
hardhat-upgrades and hardhat-deploy doesn't support it, so, in fixtures you still needed it in some specific places.